src/TSTLErrors.ts

```diff
@@ -74,9 +95,12 @@ export class TSTLErrors {
     public static UnsupportedProperty = (parentName: string, property: string, node: ts.Node) =>
         new TranspileError(`Unsupported property on ${parentName}: ${property}`, node);
 
-    public static UnsupportedForTarget = (functionality: string, version: string, node: ts.Node) =>
+    public static UnsupportedForTarget = (functionality: string, version: string | undefined, node: ts.Node) =>
```

Comment:
`Lua undefined` doesn't sound right, won't it be better to handle it in the caller? It won't complain about nullable value if it would be `this.luaTarget = this.options.luaTarget || LuaTarget.LuaJIT` in the LuaTransformer constructor. That also would avoid mutating passed options object.

Reply:
Good idea